guides/common/modules/ref_glossary-of-terms-used-in-project.adoc

```diff
@@ -0,0 +1,365 @@
+
+ifdef::satellite[]
+[[{smart-proxy-context}]]
+{SmartProxy}:: An additional server that can be used in a {ProjectName} deployment to facilitate content federation and distribution (act as a Pulp mirror), and to run other localized services (Puppet server, *DHCP*, *DNS*, *TFTP*, and more).
+{SmartProxies} are useful for {Project} deployment across various geographical locations.
+In upstream Foreman terminology, {SmartProxy} is referred to as Smart Proxy.
+endif::[]
+
+[[Catalog]]
+Catalog:: A document that describes the desired system state for one specific host managed by Puppet.
+It lists all of the resources that need to be managed, as well as any dependencies between those resources.
+Catalogs are compiled by a Puppet server from Puppet Manifests and data from Puppet Agents.
+
+[[Candlepin]]
+Candlepin:: A service within Katello responsible for subscription management.
+
+[[Compliance_Policy]]
+Compliance Policy:: Refers to a scheduled task executed on {ProjectServer} that checks the specified hosts for compliance against SCAP content.
+
+[[Compute_Profile]]
+Compute Profile:: Specifies default attributes for new virtual machines on a compute resource.
+
+[[Compute_Resource]]
+Compute Resource:: A virtual or cloud infrastructure, which {ProjectName} uses for deployment of hosts and systems.
+Examples include {oVirt}, {OpenStack}, EC2, and VMWare.
+
+[[Container]]
+Container (Docker Container):: An isolated application sandbox that contains all runtime dependencies required by an application.
+{Project} supports container provisioning on a dedicated compute resource.
+
+[[Container_Image]]
+Container Image:: A static snapshot of the container’s configuration.
+{Project} supports various methods of importing container images as well as distributing images to hosts through content views.
+
+[[Content]]
+Content:: A general term for everything {Project} distributes to hosts.
+Includes software packages (RPM files), or Docker images.
+Content is synchronized into the Library and then promoted into lifecycle environments using content views so that they can be consumed by hosts.
+
+[[Content_Delivery_Network_CDN]]
+Content Delivery Network (CDN):: The mechanism used to deliver Red{nbsp}Hat content to {ProjectServer}.
+
+[[Content_Host]]
+Content Host:: The part of a host that manages tasks related to content and subscriptions.
+
+[[Content_View]]
+Content View:: A subset of Library content created by intelligent filtering.
+Once a content view is published, it can be promoted through the lifecycle environment path, or modified using incremental upgrades.
+
+[[Discovered_Host]]
+Discovered Host:: A bare-metal host detected on the provisioning network by the Discovery plugin.
+
+[[Discovery_Image]]
+Discovery Image:: Refers to the minimal operating system based on {EL} that is PXE-booted on hosts to acquire initial hardware information and to communicate with {ProjectServer} before starting the provisioning process.
```

Comment:
Capitalization for the glossary entries is inconsistent (Discovery Image, Discovery plugin, ...). I think all should be sentence case, right?